### ISSUE TITLE
Website - Main Steps Page – Poor Color Contrast of Step Icons

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Ilab-do-good#65]
The color contrast of all the Step icons backgrounds and to the white image is between 4.3:1 and 1.88:1. Except for the darker purple one which has enough color contrast at 5.33:1. The circle icons to the light gray gear background have poor color contrast too. The darker purple one passes barely with a color contrast ratio of 4.63:1.

See Screenshot below.

![complychain - main steps - poor color contrast of icons](https://user-images.githubusercontent.com/10423487/30119831-09a1b2e0-92f5-11e7-9e65-afd397fa1283.png)

**Comment by adriennerlong** [comment]
This image has been reviewed and approved by our leadership. I'll need to check with them to determine how we want to address this issue.

**Comment by jfc3-dol** [comment]
@adriennerlong,

It's a risk that ILAB will being taking that people could complain it's not accessible to them because of the poor color contrast.

**Comment by jfc3-dol** [comment]
The issue persists over all three platforms (iOS, Android, and web).

**Comment by adriennerlong** [comment]
From ILAB: Our contract with our graphic designer has ended so this is something we are not prepared to address right now. However, we can make this a priority in the next re-vamp of the app (expected 2018) once we have a new contract with our graphic designers.

**Comment by jfc3-dol** [comment]
@adelevie,

Okay, will leave this open so we have it as a reference for the next version of ComplyChain.

**Comment by jfc3-dol** [comment]
The new Main Steps image and messages of what step a person is on now have enough color contrast for the web, iOS, and Android versions of the application.

closing issue.
